tdrs-backend/tdpservice/users/test/test_permissions.py

```diff
@@ -64,7 +64,6 @@ def test_ofa_system_admin_permissions(ofa_system_admin):
         'stts.change_stt',
         'stts.view_region',
         'stts.view_stt',
-        'users.add_user',
```

Comment:
Would like to discuss this offline.

Reply:
![image](https://user-images.githubusercontent.com/10676925/151834983-aea478b4-d224-4e92-8947-5f240b7fa2c7.png)

Reply:
it does.